src/window/window_wrapper/mod.rs

```diff
@@ -57,22 +56,7 @@ impl GlutinWindowWrapper {
         let window = self.windowed_context.window();
         if self.fullscreen {
             window.set_fullscreen(None);
-
-            // Use cached size and position
-            window.set_inner_size(LogicalSize::new(
-                self.cached_size.width,
-                self.cached_size.height,
-            ));
-            window.set_outer_position(LogicalPosition::new(
```

Comment:
Looks like this got removed. Was this intentional? I don't remember off the top of my head what the state is for cached window size

Reply:
Oh, that's fullscreen fix from earlier PR, probably i forgot to rebase.